js/apps/system/_api/gharial/APP/gharial.js

```diff
@@ -775,6 +776,43 @@ router.post('/:graph/edge/:collection', function (req, res) {
       {errorNum: errors.ERROR_GRAPH_INVALID_EDGE.code}
     );
   }
+  // check existence of _from and _to vertices
+  // _from vertex
+  try {
+    const fromCol = req.body._from.split('/')[0];
+    db[fromCol].document(req.body._from);
```

Comment:
Can't we do get rid of splitting and simply use `db._document(req.body._from)` instead?

Reply:
:+1:

Reply:
done